dotnet/src/VectorDataIntegrationTests/VectorDataIntegrationTests/CRUD/BatchConformanceTests.cs

```diff
@@ -66,4 +66,65 @@ await this.ExecuteAsync(async collection =>
             Assert.Equal("keys", ex.ParamName);
         });
     }
+
+    [ConditionalFact]
+    public Task CanInsertUpdateAndDelete_WithVectors()
```

Comment:
We can align naming conventions later, but I've gone with snake_case for the tests (see the filter tests for example); with the long sentences that test methods sometimes get to, that seems good for readability.

Note that this differs from the convention in this repo, but the vector data tests will be moving out to Microsoft.Extensions anyway later.